packages/Event_/src/index.native.tsx

```diff
@@ -73,7 +73,8 @@ function buildHitSlop({ hitSlop, ...props }: IProps) {
 class TouchableOpacity extends React.Component<IProps, void> {
 
   static defaultProps = {
-    activeOpacity: 0.2,
+    activeOpacity: 0.6,
+    onPress: null,
```

Comment:
If we need a default for `onPress` it should probably be a function like `() => null`, otherwise I believe typescript would no likey